servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -108,7 +108,7 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 		~RenderBufferDataForwardClustered();
 	};
 
-	virtual RenderBufferData *_create_render_buffer_data();
+	RenderBufferData *_create_render_buffer_data() override;
```

Comment:
I think we standardized on using both `virtual` and `override` for now.

Reply:
```
$ rg "virtual .* override" | wc -l
5143
```

Reply:
That caused compile errors for me

Reply:
(or warnings)

Reply:
I'm wrong, both work fine, will change it